airbyte-webapp/src/packages/cloud/views/auth/SignupPage/components/SignupForm.tsx

```diff
@@ -195,15 +196,27 @@ export const SignupForm: React.FC = () => {
     return yup.object().shape(shape);
   }, [showName, showCompanyName]);
 
-  return (
-    <Formik<FormValues>
-      initialValues={{
+  const [params] = useSearchParams();
+  const search = Object.fromEntries(params);
+
+  const initialValues = params.get("email")
+    ? {
+        name: search.firstname ? `${search.firstname} ${search.lastname}` : "",
+        companyName: search.company,
+        email: search.email,
```

Comment:
@letiescanciano what fields new form will have?  
If only the `email` field, then perhaps we don't need to set the rest ones...

Reply:
Because of hubspot cookies, it will have all those other hidden fields, meaning that if the user has completed that info at any time and we have that info associated with her in Hubspot, we can use it to prefill this form

Reply:
> Because of hubspot cookies, it will have all those other hidden fields, meaning that if the user has completed that info at any time and we have that info associated with her in Hubspot, we can use it to prefill this form

Oh, okay then 👍